prometheus/summary.go

```diff
@@ -54,6 +54,9 @@ type Summary interface {
 }
 
 // DefObjectives are the default Summary quantile values.
+//
+// Deprecated: DefObjectives will be removed in v0.10. The default will be a
```

Comment:
I agree. While users shouldn't really be using these in the first place, it's a breaking change and this will help quell some of the rumblings it'll cause.